tools/trusted-app-client/src/test/java/edu/emory/cci/bindaas/trusted_app_client/test/TestTrustedAppClient.java

```diff
@@ -114,14 +120,15 @@ public void testListAPIKey()
 		String url = "http://localhost:9099/trustedApplication";
 		String applicationID = "demo-id";
 		String applicationSecret = "demo-secret-key";
+		String protocol = "api_key";
```

Comment:
If we are always checking for "api_key", we do not need to check for "API_KEY" or do equalsIgnorecase(). Pls decide the right approach.

Probably just api_key is ok.

Reply:
This difference comes from the fact that the configuration.json file is as :
```json
{
"authenticationProtocol":"API_KEY"
}
```

whereas when we run the client from command line we write,
```
java -jar trusted-app-client.jar -protocol api_key
```

I have generally seen command line args to be in lowercase so that's why I followed the same here. Should I change the naming style in the config.json file?

Reply:
Got it. In that case, I recommend we leave as it is. We shouldn't give unnecessary surprises to someone who decides to upgrade Bindaas in the future.